lisa/sut_orchestrator/azure/common.py

```diff
@@ -1036,6 +1048,52 @@ def delete_file_share(
     share_service_client.delete_share(file_share_name)
 
 
+def get_diagnostic_data(
```

Comment:
It's not used outside of this module, make it private.

Reply:
updated